src/elements/RadialLoader/RadialLoader.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const RadialLoader = (props) => {
+	//Set default size
+	const width = props.width ? props.width : 20,
```

Comment:
Since this is a single SVG that we know will always be returned, and we know it should be proportional in its dimensions, this way of doing it could lead to problems (i.e., I pass in `width` of `50`, and then `height` defaults to `20`, we won't have the expected result).

Instead of accepting `height` and `width`, it'd probably be better to have a single `size` prop. You then use that single value to apply to the `height` and `width` style values.

Also, pretending that what I said above wasn't an issue, it'd be better to set these as defaults in the parameter definitions like so:

`const RadialLoader = ({ width = 20, height = 20 }) => {`

This saves you 2 variable definitions and 2 ternary operations and would achieve the same effect.